src/sonic-host-services/scripts/caclmgrd

```diff
@@ -400,6 +400,7 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
                 acl_rules = {}
 
                 for ((rule_table_name, rule_id), rule_props) in self._rules_db_info.items():
+                    rule_props = {k.upper(): v for k,v in rule_props.iteritems()}
```

Comment:
As with the previous problem, I you also have `.iteritems()` here, which is no longer available in Python 3. It should be replaced with `.items()`.

Reply:
Addressed in next commit